data/mods/Magiclysm/traps.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "type": "trap",
+    "id": "tr_bear",
+    "name": "bear trap",
+    "looks_like": "tr_beartrap",
```

Comment:
Traps don't support "looks_like". This causes the test to fail and will cause a debug message when the game starts.

Reply:
ahh bummer. i'll remove it, though i definitely think it should gain this functionality